proxy/http/Http1Transaction.h

```diff
@@ -37,6 +37,7 @@ class Http1Transaction : public ProxyTransaction
 
   ////////////////////
   // Methods
+  void init_transaction(Http1ClientSession *ssn); // overload, not override
```

Comment:
I'm fine with overloading, but why do we want this method? I think we can pass Http(1|2|3)ClientSesion to constructors.